Cargo.toml

```diff
@@ -1,30 +1,19 @@
 [package]
-name = "cargo-edit"
-authors = ["Without Boats <lee@libertad.ucsd.edu>",
-           "Pascal Hertleif <killercup@gmail.com>",
-           "Sebastian Garrido <sebasgarcep@gmail.com>",
-           "Jonas Platte <mail@jonasplatte.de>"]
-
-version = "0.1.6"
-
+authors = ["Without Boats <lee@libertad.ucsd.edu>", "Pascal Hertleif <killercup@gmail.com>", "Sebastian Garrido <sebasgarcep@gmail.com>", "Jonas Platte <mail@jonasplatte.de>"]
```

Comment:
What happened to this file? Did you use cargo add? :trollface: 

No, seriously, can you try redo the changes to this file? I think you only removed rustc_serialize, added serde, and updated docopt.